rovercode/drivers/dummy_binary_sensor.py

```diff
@@ -15,6 +15,6 @@ def __init__(self):
         """Create the dummy sensor."""
         LOGGER.info("Created dummy binary sensor")
 
-    def is_high(self):
+    def is_high(self):  # noqa
```

Comment:
What did it not like about this?

Reply:
It said that it didn't need `self` so it could be a function instead of a method. But since it's a dummy, I liked leaving `self` for the consistency.